test/functional/sqlsrv/sqlsrv_errors.phpt

```diff
@@ -119,7 +119,7 @@ sqlsrv_close returns true even if an error happens.
     echo "Test successfully done.\n";
 ?>
 --EXPECTF--
-Warning: sqlsrv_close() expects parameter 1 to be resource, boolean given in %Ssqlsrv_errors.php on line %x
+Warning: sqlsrv_close() expects parameter 1 to be resource, bool%S given in %Ssqlsrv_errors.php on line %x
```

Comment:
PHP 7.3 changes he output to just `bool` and `int`?

Reply:
yes